src/test/java/com/hyunec/cosmicbaseballinit/vo/PitchResultTest.java

```diff
@@ -0,0 +1,19 @@
+package com.hyunec.cosmicbaseballinit.vo;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+class PitchResultTest {
+
+    @Test
+    @DisplayName("투구의 결과 enum 클래스의 name 값이 잘 호출 되는지 테스트")
+    void getResultName() {
+        String name = PitchResult.HIT.getResultName();
```

Comment:
```suggestion
        String name = PitchResult.HIT.name();
```

`nit` 혹시 enum name 을 그대로 가져오려는 목적이라면 이런 방법도 있습니다.